package.json

```diff
@@ -25,7 +25,7 @@
     "generate:nerdctl-stub": "powershell scripts/windows/generate-nerdctl-stub.ps1",
     "build": "node scripts/ts-wrapper.js scripts/build.ts",
     "sign": "node scripts/ts-wrapper.js scripts/sign.ts",
-    "wix": "node scripts/ts-wrapper.js scripts/wix.ts",
+    "wix": "node scripts/ts-wrapper.js scripts/wix.ts -ext WixUIExtension -ext WixUtilExtension",
```

Comment:
These should already be in the script; are they missing? (And if they are, it'd be better to put them there — `npm run wix` is only a development aid, the real build wouldn't be using it.)

https://github.com/rancher-sandbox/rancher-desktop/blob/0a26306847d0afedb2f0c98cff02c21149159892/scripts/lib/installer-win32.tsx#L100-L101

Reply:
No, I didn't dive down to see where the command-line is